geometry/geometry_config.py

```diff
@@ -70,11 +70,15 @@
     shieldName = None
 if "SND" not in globals():
     SND = True
+if "SNDDesign" not in globals():
+    SNDDesign = 1
```

Comment:
What is the difference between this and the previous NuTauTargetDesign flag?
I would avoid duplicates since it creates confusion, so if we keep this one we should remove the old one.

Reply:
I'd vote in favour of removing NuTauTargetDesign:
- The SND is **not** exclusively a $\nu_\tau$ detector, and one shouldn't need to have studied the pre-ECN3 history to use the script
- The "Target" part might cause confusion. Not everyone might expect the scattering detector be called target, even if if technically is a secondary target for BSM and neutrinos.

Maybe `SND_design` or `SND_DESIGN` would be better than `SNDDesign` to follow common python naming conventions.